### DISCUSSION TITLE
Playtesting: Bug or feature?

### DISCUSSION THREAD

**Discussion by toastmartin** [diasurgical/DevilutionX#6880] [Q&A]
I was able to do some extensive playtesting over the holydays :-) and noticed some interesting points. Regarding these points, I am interested in the oppinion of the project leaders and the other forum folks: Bug or Feature? (As always: Great game overall, these are small points.)

a) The learnable Apocalypse spell starts at a mana cost of 90. The cost as well as the damage are then independent of the spell level. This was unexpected for me. Jarulf writes that Apocalypse should cost 150 Mana at level 1 and 6 mana less for each ppell level down to a minimum of 90. Is this intentional?
b) The Gargoyles in the caves (not sure if "Winged Demon" or "Gargoyle") seem to have a "stand up"-sequence that is much slower than the rest of their movement. It feels slower than in the original game.
c) Weapon oils (damage and +hit, probably armor oils too) seem to be permanent over different games. This is different from the original game. Is it intentional? What were the reasons behind this descision? That can make weapons pretty overpowered if you are playing for oils long enough.

**Comment by DakkJaniels** [comment]
a) assuming you are using a sorcerer in Hellfire, you get a 50% reduction in mana cost, bringing you to the minimum mana cost. No different from original.
b) not sure
c) that's intentional

**Comment by StephenCWills** [comment]
We've talked about the Gargoyles pretty extensively in #2917.

I wasn't around when the decision was made about oils, but my understanding is that clearing the effects between games was probably unintentional. Likely there were a few factors that were considered, like the fact that durability doesn't get reverted, the fact that Hellfire was SP-only, some of the changes Sierra made to the `ItemPack` struct, etc.

**Reply by AJenbo** [reply]
Also the way they had done things, making the oils permanent would have broken how they handled imported Diablo heros in to Hellfire.

**Reply by StephenCWills** [reply]
The more I think about it, the more I think the `ItemPack` struct must have been the same. Maybe I'm thinking of the `TCmd*Item` structs rather than the `ItemPack` struct.
